db/app.js

```diff
@@ -15,6 +15,8 @@ app.get("/api/reviews/", getReviews);
 
 app.get("/api/reviews/:review_id/", getReview);
 
+app.get("/api/reviews/:review_id/comments/", getCommentsForReview);
+
 app.use((req, res, next) => {
   res.status(404).send({msg: "path not found"})
 })
```

Comment:
have a look at how you might create a generic custom error handler that checks if there's err.status && err.msg, and will send whatever that status and message is. That way you can define status and msg when you reject a promise.